bridge/.gitignore

```diff
@@ -58,3 +58,4 @@ e2e/screenshots/*
 !cypress/fixtures/files/*.pem
 *.crt
 *.key
+/documentation.json
```

Comment:
It looks like `/documentation.json` belongs to the group `# certificates`.
Could you add a new comment `# storybook` and move this below that.